src/server/vfs/mac/CMakeLists.txt

```diff
@@ -2,56 +2,56 @@ find_package(Qt6)
 
 # Target
 add_library("${libsyncengine_NAME}_vfs_mac" SHARED
-    vfs_mac.h vfs_mac.cpp
-    litesyncextconnector.mm
-    ../../../libcommonserver/vfs.h ../../../libcommonserver/vfs.cpp
-    ../../../libcommonserver/plugin.h ../../../libcommonserver/plugin.cpp
-    ../../../common/filepermissionholder.h ../../../common/filepermissionholder.cpp
-    ../../../common/filesystembase.h ../../../common/filesystembase.cpp
+        vfs_mac.h vfs_mac.cpp
```

Comment:
Your IDE is formatting `CMakeList.txt` files. That's fine if the output looks better. But we could do all changes of this sort in a separate PR.

Reply:
I have deactivated the auto formatting on CMake files